bindings/rust/evmc-vm/src/lib.rs

```diff
@@ -308,4 +480,99 @@ mod tests {
             }
         }
     }
+
+    unsafe extern "C" fn get_dummy_tx_context(
+        _context: *mut ffi::evmc_context,
+    ) -> ffi::evmc_tx_context {
+        ffi::evmc_tx_context {
+            tx_gas_price: ffi::evmc_uint256be { bytes: [0u8; 32] },
+            tx_origin: ffi::evmc_address { bytes: [0u8; 20] },
+            block_coinbase: ffi::evmc_address { bytes: [0u8; 20] },
+            block_number: 42,
+            block_timestamp: 235117,
+            block_gas_limit: 105023,
+            block_difficulty: ffi::evmc_uint256be { bytes: [0xaa; 32] },
+        }
+    }
+
+    unsafe extern "C" fn get_dummy_code_size(
+        _context: *mut ffi::evmc_context,
+        _addr: *const ffi::evmc_address,
+    ) -> usize {
+        105023usize
```

Comment:
Nitpicking but can you use `105023 as usize`?

Never liked these compound forms in C++ either.